src/Microsoft.AspNetCore.Razor.Sdk/build/netstandard2.0/Sdk.CurrentVersion.targets

```diff
@@ -0,0 +1,395 @@
+﻿<Project>
+  <!-- 
+    Targets supporting Razor MSBuild integration. Contain support for generating C# code using Razor
+    and including the generated code in the project lifecycle, including compiling, publishing and producing
+    nuget packages.
+  -->
+
+  <!--
+    This is a hook to import a set of targets before the Razor targets. By default this is unused.
+  -->
+  <Import Project="$(CustomBeforeRazorSdkTargets)" Condition="'$(CustomBeforeRazorSdkTargets)' != '' and Exists('$(CustomBeforeRazorSdkTargets)')"/>
+
+  <PropertyGroup>
+    <MicrosoftAspNetCoreRazorSdkTargetsImported>true</MicrosoftAspNetCoreRazorSdkTargetsImported>
+  </PropertyGroup>
+
+  <!--
+    Razor defines two primary targets:
+      'RazorGenerate' - which updates generated code
+      'RazorCompile' - compiles an assembly from generated code
+
+    Use these properties and targets to attach behavior to the corresponding phase.
+  -->
+  <PropertyGroup>
+    <PrepareForRazorGenerateDependsOn>
+      ResolveRazorGenerateInputs;
+      ResolveAssemblyReferenceRazorGenerateInputs;
+      ResolveTagHelperRazorGenerateInputs
+    </PrepareForRazorGenerateDependsOn>
+
+    <RazorGenerateDependsOn>
+      PrepareForRazorGenerate;
+      _EnsureRazorCompilerReferenced;
```

Comment:
Added this as a guard against the user invoking `/t:RazorCoreGenerate` without referencing the package.

Reply:
The other thing you could do is define the target here to always report an error, and then override with imports.